tools/src/common/mod.cpp

```diff
@@ -55,6 +57,9 @@ tMod::tMod(const std::string &szFileName)
 
 		// Data read successfully, fill sample info
 		tSample Sample;
+		for (uint8_t i = 0; i < SAMPLE_NAME_SIZE; ++i){ 
+			szSampleNameRaw[i] = toupper(szSampleNameRaw[i]); // sample name to uppercase, to avoid duplicates
```

Comment:
this will work, but the side effect is that the sample names will all be in uppercase in resulting mods. Dunno if it's good - perhaps it's better to leave the case as is and just do name comparison between samples on temporarily uppercased copies.

Also, doesn't it produce error/warning by writing `toupper()` instead of `std::toupper()`? It's better to write it this way.